### PR TITLE
Add default value for data version in CLI

### PR DESCRIPTION
**Pull request by Shubhamr837**
Default dataVersion is set to 2 for the CLI .
should fix https://github.com/odk-x/tool-suite-X/issues/197

### CODE REVIEW

**Comment by wbrunette** [comment]
runtests

File: src/main/java/org/opendatakit/suitcase/ui/SuitcaseCLI.java [hunk 1]
```diff
@@ -326,8 +324,12 @@ private Operation parseArgs(String[] args, Options options) {
       
       updateLogPath = line.getOptionValue(UPDATE_LOG_PATH_OPT);
 
-      version = line.getOptionValue(DATA_VERSION_OPT);
-
+      if (!line.hasOption(DATA_VERSION_OPT)) {
+        version=default_data_version;
+      }
+      else {
+        version = line.getOptionValue(DATA_VERSION_OPT);
```
Comment (linl33) [inline src/main/java/org/opendatakit/suitcase/ui/SuitcaseCLI.java:331]: You could supply the default value directly to `getOptionValue` see an example on https://github.com/odk-x/suitcase/pull/49/files#diff-9851d999d0c7a8e7293d5cbb8cf98908b843b59ce1485c203537e99dbbcb9684L325

File: src/main/java/org/opendatakit/suitcase/ui/SuitcaseCLI.java [hunk 2]
```diff
@@ -78,6 +79,7 @@
   private boolean scanFormatting;
   private boolean extraMetadata;
   private boolean force;
+  private String default_data_version = "2";
```
Comment (linl33) [inline src/main/java/org/opendatakit/suitcase/ui/SuitcaseCLI.java:82]: Please make this a static final constant.

**Review by linl33 — CHANGES_REQUESTED** [review]
Thanks for sending in the pull request!

**Comment by linl33** [comment]
runtests

**Comment by Shubhamr837** [comment]
Thanks for the suggestions . I have implemented the changes .
Please Review If any other changes are required .

**Comment by linl33** [comment]
runtests

**Review by linl33 — APPROVED** [review]

**Pull request merged**